go.mod

```diff
@@ -46,7 +46,9 @@ replace (
 	// one of our direct dependencies begins using containerd v1.4.0+
 	github.com/containerd/containerd => github.com/containerd/containerd v1.4.3
 	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
+	github.com/operator-framework/api => github.com/camilamacedo86/api v0.0.0-20210324013927-c04e53df9785
```

Comment:
/hold

It is using my fork. We cannot merge it until we are able to get the changes merged in the api.